integration/teardown/teardown.go

```diff
@@ -3,40 +3,143 @@
 package teardown
 
 import (
+	"context"
+	"fmt"
+
+	"github.com/giantswarm/backoff"
 	"github.com/giantswarm/e2e-harness/pkg/framework"
 	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
 
-	"github.com/giantswarm/kvm-operator/integration/env"
+	"github.com/giantswarm/kvm-operator/integration/utils"
```

Comment:
FYI it is a kind of anti pattern in go to call packages `util` or something abstract like that by convention. We should try to be more specific. Such names are a sign of weak design.

Reply:
So separating the file into two packages `rangepool` and `ipam` would make more sense in this case?

Reply:
When this is what the package essentially is about, then yes. It would at least be better than `utils`.